src/main.tsx

```diff
@@ -12,6 +12,20 @@ import './styles/fonts.css'
 import { AnimatePresence } from 'framer-motion'
 import ConfiguredRoutes from './routes'
 import { Modals } from './Modals'
+import * as Sentry from '@sentry/react'
+
+Sentry.init({
+  dsn: 'https://96f54440c19eb6650fd5a502de09a306@o1095249.ingest.sentry.io/4505754150043648',
+  integrations: [
+    new Sentry.BrowserTracing({
+      tracePropagationTargets: ['localhost', 'https://web3inbox-dev-hidden.vercel.app']
```

Comment:
Do we want to report on `localhost`? Generally produces a lot of noise from dev testing/experimentation